test/test_queries.cpp

```diff
@@ -64,8 +69,12 @@ TEST_CASE("Compute parsing function")
     }
     WHEN("With terms, stopwords, and stemmer")
     {
-        auto parse =
-            resolve_query_parser(queries, lexfile.string(), stopwords_filename.string(), "porter2");
+        auto parse = resolve_query_parser(
+            queries,
+            std::make_unique<WhitespaceTokenizer>(),
```

Comment:
And again: Just checking that you actually want `<WhitespaceTokenizer>` here?

Reply:
Upon further checking I think you do, so feel free to ignore these :-)

Reply:
Yeah, the truth is it doesn't really matter here, this is not what's mainly being tested. Both tokenizers will work the same way. I added a test that does check the tokenizer works as expected, but for the rest I just opted for the simple one. But it makes no difference, really.